docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "2.1"
+
+services:
+
+  osqa:
+    build:
+      context: .
+    environment:
+      OSQA_DB_USER: osqa
+      OSQA_DB_PASSWORD: osqa
+      OSQA_DB_NAME: osqa
+      OSQA_DB_HOST: postgres
+
+    ports:
+      - 8080:8080
+    depends_on:
+      postgres:
+        condition: service_healthy
+      memcached:
+        condition: service_started
+
+  postgres:
+    image: postgres:10
```

Comment:
Why postgres 10 when we currently use 14 and 15 is now available?

Reply:
I have gone for what we are currently using in production. I am unsure about OSQA's postgres compatibility.

Reply:
You haven't though - as I said we are using 14 in production!

Reply:
shenron is running `postgresql-10 10.22-1.pgdg20.04+1`

```
/usr/lib/postgresql/10/bin/postgres -D /var/lib/postgresql/10/main -c config_file=/etc/postgresql/10/main/postgresql.conf
```

Reply:
Only the client tools are postgres `14`

Reply:
Damn you're right... I could have sworn I had upgraded all the old postgres instances but I see that one is not being monitored in prometheus which is why I missed it so that's two things to fix!

Reply:
Updated to postgres 14 now.